components/Banner.tsx

```diff
@@ -30,19 +30,21 @@ const Banner = ({ bannerHeight, setBannerHeight }: {
 					return `h-${bannerHeight}`;
 				})
 				break;
-			case "incompatible-browser":
-				setBannerHeight(() => {
-					const bannerHeight = 24;
-					setBannerHTML((<>
-						<Image src={chromeLogo} alt="chrome extension logo" className={`inline m-6 mr-8 border border-white rounded-full w-${bannerHeight - 12}`}></Image>
-						<p className='text-center font-bold text-xl w-fit sm:max-w-1/2 h-fit my-auto'>
-							Vibinex is only supported in Chromium browsers<br />
-							<span className='text-sm font-normal'>Google Chrome, Microsoft Edge, Opera, Chromium, Brave etc.</span>
-						</p>
-					</>))
-					return `h-${bannerHeight}`;
-				})
-				break;
+            case "incompatible-browser":
+                setBannerHeight(() => {
+                    const bannerHeight = 24;
+                    setBannerHTML((<>
+                        <div className="flex items-center m-6">
+                            <Image src={chromeLogo} alt="chrome extension logo" className="w-12 h-12 border border-white rounded-full"></Image>
```

Comment:
I had written `w-${bannerHeight-12}` so that if we change the banner height in the future, we don't have to separately change the width of this element.

I don't think you should have removed the right margin (`mr-8`). It might be looking better on your screen's dimensions, but I added that margin based on different screen sizes. Please test them (using the "responsive" option in the browser's dev tools) and let me know if you still recommend removing this right margin.

Reply:
I've changed the width according to the bannerHeight and also added the right margin.